core/src/main/java/org/elasticsearch/rest/action/search/RestMultiSearchAction.java

```diff
@@ -91,6 +91,9 @@ public RestMultiSearchAction(Settings settings, RestController controller, Clien
     @Override
     public void handleRequest(final RestRequest request, final RestChannel channel, final Client client) throws Exception {
         MultiSearchRequest multiSearchRequest = new MultiSearchRequest();
+        if (request.hasParam("max_parallel_searches")) {
```

Comment:
Nit: I think that name of the request parameter should reflect concurrency rather than parallelism.